subscription-engine/kafka-ws/README.md

```diff
@@ -335,9 +338,12 @@ redis.host| Redis host | kafka-ws-redis | string | KAFKA_WS_REDIS_HOST
 redis.port | Redis port | 6379 | integer | KAFKA_WS_REDIS_PORT
 redis.db | Redis database | 1 | integer | KAFKA_WS_REDIS_DB
 redis.connect_timeout | Redis Connection timeout | 3600000 | integer | KAFKA_WS_REDIS_CONNECT__TIMEOUT
-redis.strategy.connect.after | Redis strategy connection timeout (ms) | 5000 | integer | KAFKA_WS_REDIS.RECONNECT.AFTER
+redis.strategy.connect.after | Redis strategy connection timeout (ms) | 5000 | integer | KAFKA_WS_REDIS_RECONNECT_AFTER
+redis.healthcheck.ms | Specifies how often it is to check if it is possible to communicate with the redis in milliseconds. | 30000 | integer | KAFKA_WS_REDIS_HEALTHCHECK_MS
+redis.healthcheck.timeout.ms | the timeout to wait if the service can verify if it is health, if the timeout occurs the service is considered unhealthy. (ms) | 30000 | integer | KAFKA_WS_REDIS_HEALTHCHECK_TIMEOUT_MS
```

Comment:
5000?

Reply:
I made this a timeout in case I didn't receive anything from the redis ... This solves the test q @ctossato  did ... It was the only way because Redis stopped responding

Reply:
https://github.com/dojot/dojot/issues/2094#issuecomment-827787909

Reply:
Maybe I should have just not solved this issue ... To be honest I didn't think the test made sense ...

Reply:
or are you just asking about the 5000? hahaha

Reply:
Sorry for the previous short text!
I'm just point that here the default value is 30000, but in the default file is 5000.